Readme.md

```diff
@@ -5,12 +5,32 @@
 <!-- default badges end -->
 # How to bind the Data Grid to a strongly typed collection loaded asynchronously
 
-This example consists of the following pages:
-* [Data Property](./CS/DataBindingSamples/Pages/DataProperty.razor) - Assigns a strongly typed collection to the [Data](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.Data) property and uses an asynchronous task to initialize the collection. In this case, the Data Grid page is rendered twice.
-* [DataAsync Property](./CS/DataBindingSamples/Pages/DataAsyncProperty.razor) - Uses the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property to load a strongly typed collection asynchronously. Specifies a key data field to identify a data source instance.
-* [DataAsync: Cancellation Token](./CS/DataBindingSamples/Pages/DataAsyncProperty.CancellationToken.razor) - Uses the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property to load a data collection asynchronously and processes a cancellation token to prevent further execution for the already disposed Data Grid.
-* [DataAsync: Cache](./CS/DataBindingSamples/Pages/DataAsyncProperty.Cache.razor) - Caches the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) function. This is helpful if the function is time- or resource-consuming.
+You can bind the Data Grid component to a collection that is loaded synchronously and asynchronously.
 
-For more information, refer to the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property description.
+![Data Grid: Data Binding Main Page](images/example-content.png)
+
+In this example, you can navigate between four approaches that allow you to bind the DxDataGrid to a asynchronously loaded collection:
+* [Data Property](./CS/DataBindingSamples/Pages/DataProperty.razor): Assigns a strongly typed collection to the [Data](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.Data) property and uses an asynchronous task to initialize the collection. The Data Grid page is rendered twice.
+* [DataAsync Property](./CS/DataBindingSamples/Pages/DataAsyncProperty.razor): Uses the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property to load a strongly typed collection asynchronously. Specifies a key data field to identify a data source instance.
+* [DataAsync: Cancellation Token](./CS/DataBindingSamples/Pages/DataAsyncProperty.CancellationToken.razor): Uses the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property to load a data collection asynchronously and processes a cancellation token to prevent further execution for the already disposed Data Grid.
```

Comment:
```suggestion
* [DataAsync: Cancellation Token](./CS/DataBindingSamples/Pages/DataAsyncProperty.CancellationToken.razor): Uses the [DataAsync](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.DataAsync) property to load a data collection asynchronously and processes a cancellation token to prevent further execution for the disposed Data Grid.
```